Solutions/SymantecProxySG/Package/createUiDefinition.json

```diff
@@ -60,14 +60,14 @@
             "name": "dataconnectors1-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This solution installs the data connector for ingesting logs from Symantec ProxySG in syslog format. After installing the solution, configure and enable this data connector by following guidance in Manage solution view"
+              "text": "This Solution installs the data connector for SymantecProxySG. You can get SymantecProxySG Syslog data in your Microsoft Sentinel workspace. After installing the solution, configure and enable this data connector by following guidance in Manage solution view."
             }
           },
           {
             "name": "dataconnectors-parser-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "The solution also installs a parser that transforms ingested data. The transformed logs can be accessed using the SymantecProxySG Kusto Function alias."
+              "text": "The Solution installs a parser that transforms the ingested data into Microsoft Sentinel normalized format. The normalized format enables better correlation of different types of data from different data sources to drive end-to-end outcomes seamlessly in security monitoring, hunting, incident investigation and response scenarios in Microsoft Sentinel."
```

Comment:
Hi @v-prasadboke please remove the changes for data-connector and parser in this, thanks!